### PR TITLE
Translate Adding a Service Worker

### PR DESCRIPTION
**Pull request by zweimach**
Please review my translation. Thank you.

**Edit:** This PR seems to be a duplicate of #18. I did it without confirming first. Please forgive my rudeness.

### CODE REVIEW

File: docs/docs/add-a-service-worker.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
 ---
-title: Adding a Service Worker
+title: Menambahkan Service Worker
 ---
 
-## What is a service worker
+## Apa itu *service worker*
 
-A service worker is a script that your browser runs in the background, separate from a web page, opening the door to features that don't need a web page or user interaction. They increase your site availability in spotty connections, and are essential to making a nice user experience.
+*Service worker* adalah *script* yang dijalankan oleh peramban web anda di balik layar, terpisah dengan laman web, membuka pintu untuk fitur-fitur yang tidak memerlukan laman web atau interaksi dengan pengguna. Hal ini meningkatkan ketersedian situs anda pada koneksi yang kurang lancar, dan sangat penting untuk menciptakan pengalaman pengguna yang baik.
```
Comment (coroo) [inline docs/docs/add-a-service-worker.md]: Please help to change laman to halaman

**Review by coroo — CHANGES_REQUESTED** [review]
Hi, please help us to change some comment above before we merge it

**Comment by coroo** [comment]
Its ok @zweimach . For this one, I think we need to wait from previous commit first before I can merge yours

**Comment by zweimach** [comment]
@coroo, thank you for your consideration.

**Review by coroo — APPROVED** [review]

**Pull request merged**